src/pages/index.module.scss

```diff
@@ -10,13 +10,22 @@ $list-max-width: 80rem;
     // TODO: change this to `$tablet-padding` when we remove banner
     padding-block-start: calc($tablet-padding + var(--banner-height));
   }
+
   padding-block-end: var(--spacing-small);
   padding-inline-start: 0;
   padding-inline-end: 0;
 }
 
-.flow > .flowItem {
+.loadingContainer {
```

Comment:
This seems to only be used once inside `src/pages/reading-goal/index.tsx` so I feel it's more local than global.